WordPress/Classes/Utility/Editor/GutenbergOnboardingRollout.swift

```diff
@@ -1,9 +1,26 @@
 /// This structs helps encapsulate logic related to Gutenberg editor onboarding rollout phases.
 ///
 struct GutenbergOnboardingRollout {
-    private let phasePercentage = 0
+    private let phasePercentage = 50
 
     func isUserIdInPhaseRolloutPercentage(_ userId: Int) -> Bool {
-        return userId % 100 >= (100 - phasePercentage) || BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest]
+        return convertUserIdToRank(userId: String(userId)) >= (100 - phasePercentage) || BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest]
+    }
+
+    func convertUserIdToRank(userId: String) -> Int {
+        let key = userId + "can_view_editor_onboarding"
+        let inputString = key.replacingOccurrences(of: "-", with: "")
+        return abs(inputString.djb2hash) % 100
```

Comment:
I notice the `abs` call here. Is it expected that user IDs themselves may be negative (which is what I encountered and captured in image below) or that the hashed value may be negative? It seemed odd to me that the user ID itself was negative, but I don't know if it matters. 

<img width="958" alt="negative-user-id" src="https://user-images.githubusercontent.com/438664/123432070-98786a00-d58f-11eb-9b83-ccad82733fb4.png">

Reply:
I don't think it matters since we only use the absolute value to calculate the rank either way.